stdlib/2and3/socket.pyi

```diff
@@ -713,7 +713,7 @@ if sys.version_info >= (3, 7):
 
 def create_connection(address: Tuple[Optional[str], int],
                       timeout: Optional[float] = ...,
-                      source_address: Tuple[Union[bytearray, bytes, Text], int] = ...) -> socket: ...
+                      source_address: Optional[Tuple[Union[bytearray, bytes, Text], int\] = ...) -> socket: ...
```

Comment:
These seems to be an extraneous backslash here.

Reply:
Ah sorry, the backslash was probably supposed to be a closing bracket. So this is missing now. :)

Reply:
Sorry about that.  Should be better now.